packages/core/data-transfer/package.json

```diff
@@ -53,14 +60,14 @@
     "ws": "8.13.0"
   },
   "peerDependencies": {
-    "@strapi/strapi": "^4.13.3"
+    "@strapi/strapi": "^4.13.6"
```

Comment:
I think we can delete the peerDependencies now, it seems to build fine without it now that we've moved types around.

Reply:
I tried but this break the e2e tests because yarn link isn't happy so let's keep it for now. I actually think we might want to remove strapi as a direct dependency later